base/sys_info_starboard.cc

```diff
@@ -31,7 +31,7 @@ int SysInfo::NumberOfEfficientProcessorsImpl() {
 }
 
 size_t SysInfo::VMAllocationGranularity() {
-  return 0;
+  return 4096U;
```

Comment:
Recommend adding a comment here as to why a constant/stub value is fine, e.g. we don't expect this to impact any production code ? Maybe ? 

Perhaps as with AmountOfTotalDiskSpace() below it should have NOTIMPLEMENTED just in case ? Although if this would cause useless log spam i'm not sure that would add any value either.